modules/servicetemplate/src/main/java/org/eclipse/xpanse/modules/servicetemplate/config/CspAutoApproveConfig.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.servicetemplate.config;
+
+import org.eclipse.xpanse.modules.models.common.enums.Csp;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+/**
+ * Configuration for CSP auto approve service templates.
+ */
+@Component
+public class CspAutoApproveConfig {
```

Comment:
@baixinsui i would avoid such common class for all CSPs. 
can we instead make this as a plugin interface method -> the method returns the config parameter.